tensorflow_encrypted/protocol/pond.py

```diff
@@ -426,6 +426,24 @@ def sum(self, x, axis=None, keepdims=None):
     def reduce_sum(self, x, axis=None, keepdims=None):
         return self.sum(x, axis, keepdims)
 
+    @memoize
+    def where(self, x):
```

Comment:
maybe add a type here, as it will certainly fail for anything the public tensors :D